x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/sequence/TumblingWindow.java

```diff
@@ -127,7 +133,10 @@ public TumblingWindow(
         List<SequenceCriterion> criteria,
         SequenceCriterion until,
         SequenceMatcher matcher,
-        List<List<Attribute>> listOfKeys
+        List<List<Attribute>> listOfKeys,
+        boolean allowPartialSearchResults,
```

Comment:
Given that [`allowPartialSequenceResults` depends on `allowPartialSearchResults` being `true`](https://github.com/elastic/elasticsearch/pull/116388/files#diff-efd7c22da9196006301656246c0b7bd07626fbed7107d8104f710fb8a115835eR117), I am surprised to see no checks here and `allowPartialSequenceResults` just being used. Can you point me to where we make sure that `allowPartialSequenceResults` is valid **if** `allowPartialSearchResults` has the right value?

Reply:
There is no specific check, because in case of partial shard failures the query would fail much earlier. The final result will be an error 400.